manifests/0000_50_olm_05-subscription.crd.yaml

```diff
@@ -9,33 +9,33 @@ spec:
   group: operators.coreos.com
   version: v1alpha1
   versions:
-  - name: v1alpha1
-    served: true
-    storage: true
+    - name: v1alpha1
```

Comment:
Something doesn't look right with this file with all extra spaces at the start of these lines. They shouldn't be there.